packages/components/src/components/button/_button.scss

```diff
@@ -118,9 +118,9 @@
     @include button-theme(
       transparent,
       transparent,
-      $interactive-04,
+      $link-01,
       $hover-ui,
-      $interactive-04,
+      $icon-01,
```

Comment:
the icon also needs to be `link-01` sorry that wasn't clear.
![image](https://user-images.githubusercontent.com/11670886/61004989-e6592080-a32c-11e9-933c-509f05219084.png)